packagingtest/jessie/sshd/Dockerfile

```diff
@@ -27,6 +27,10 @@ RUN echo -e "#!/bin/sh\nexit 101\n" > /usr/sbin/policy-rc.d && \
     apt-get -y install gdebi-core sshpass cron \
       netcat net-tools
 
+# install locales package and set default locale to 'UTF-8' for the test executiong environment
+RUN apt-get -y update && apt-get -y install locales && \
```

Comment:
`apt-get update` is redundant here, since it's done previously: 
https://github.com/StackStorm/st2-dockerfiles/pull/37/files#diff-6793958f7640346527c3e495e1469d9fR11